Tests/SentryTests/SentryTransactionTests.swift

```diff
@@ -2,28 +2,45 @@ import XCTest
 
 class SentryTransactionTest: XCTestCase {
     let someTransactionName = "Some Transaction"
+    let someOperation = "Some Operation"
```

Comment:
`l`: Can we please increase the code coverage to 100% for SentryTransaction. I know it's only getters, but still, it's worth the effort.

Reply:
xCode says coverage is 100% now, I don't know how to see it here.

Reply:
Codecov usually adds a coverage report here. You can see it above or you can take a look here https://codecov.io/gh/getsentry/sentry-cocoa. I don't really know why it is not updating the report in this PR after you pushed new changes.